tools/flang2/flang2exe/lldebug.cpp

```diff
@@ -3141,9 +3144,15 @@ lldbg_create_cmblk_gv_mdnode(LL_DebugInfo *db, LL_MDRef cmnblk_mdnode,
   subscripts_mdnode = llmd_finish(mdb);
   type_mdnode = lldbg_create_array_type_mdnode(
       db, ll_get_md_null(), 0, sz, align, elem_type_mdnode, subscripts_mdnode);
+#ifdef FLANG_LLVM_EXTENSIONS
+  if (ll_feature_debug_info_ver70(&db->module->ir) && !XBIT(183, 0x40000000))
```

Comment:
Is there a matching llvm change that needs to be pushed?  Or has it been pushed?